src/backend/joanie/core/models/products.py

```diff
@@ -262,6 +263,34 @@ class Meta:
     def __str__(self):
         return f"Order {self.product} for user {self.owner}"
 
+    def generate_invoice(self):
+        """Generate a pdf invoice for an order"""
+        vat = settings.VAT
+        mount_ht = float(self.product.price)
+        vat_mount = mount_ht * vat / 100
```

Comment:
```suggestion
        amount_ht = float(self.product.price)
        vat_amount = amount_ht * vat / 100
```

Reply:
I think you can't use float for money. You should use Decimal to have fixed point figures.

Reply:
Thanks @jbpenrath!  
:D `mount` instead of `amount` 
I need to sleep!